packages/graph-explorer/src/core/StateProvider/displayEdge.ts

```diff
@@ -0,0 +1,154 @@
+import { Edge, EdgeId, VertexId } from "@/@types/entities";
+import { selector, selectorFamily, useRecoilValue } from "recoil";
+import { textTransformSelector } from "@/hooks/useTextTransform";
+import {
+  edgeTypeConfigSelector,
+  vertexTypeAttributesSelector,
+  vertexTypeConfigSelector,
+} from "../ConfigurationProvider/useConfiguration";
+import { edgesAtom, edgesSelectedIdsAtom } from "./edges";
+import {
+  DisplayAttribute,
+  getSortedDisplayAttributes,
+} from "./displayAttribute";
+import {
+  DisplayEdgeTypeConfig,
+  displayEdgeTypeConfigSelector,
+} from "./displayTypeConfigs";
+import { queryEngineSelector } from "../connector";
+
+/** Represents an edge's display information after all transformations have been applied. */
+export type DisplayEdge = {
+  entityType: "edge";
+  id: EdgeId;
+  displayId: string | null;
+  displayTypes: string;
+  typeConfig: DisplayEdgeTypeConfig;
+  source: EdgeVertex;
+  target: EdgeVertex;
+  attributes: DisplayAttribute[];
+};
+
+type EdgeVertex = {
+  id: VertexId;
+  displayId: string;
+  displayTypes: string;
+};
+
+/** Maps all `Edge` instances in the graph canvas to `DisplayEdge` instances. */
+export function useDisplayEdgesInCanvas() {
+  return useRecoilValue(displayEdgesInCanvasSelector);
+}
+
+/** Finds the `DisplayEdge` instance for a give `EdgeId` within the edges added to the graph canvas. */
+export function useDisplayEdge(id: EdgeId) {
```

Comment:
This function seems unused - is it meant to be used in a future PR?

Reply:
Yes, but it is super easy to re-add. Removing for now.